scipy/integrate/_ivp/ivp.py

```diff
@@ -358,6 +358,15 @@ def solve_ivp(fun, t_span, y0, method='RK45', t_eval=None, dense_output=False,
     min_step : float, optional
         The minimum allowed step size for 'LSODA' method.
         By default `min_step` is zero.
+    catch_exceptions : bool, optional
+        Whether to catch exceptions raised during calls to `solver.step()`. 
```

Comment:
The markup convention is double ticks for "code fragments" and single ticks to reference a parameter.

Reply:
More significant. Shouldn't it be placed after `args=None` argument? It means that this argument is applicable to `solve_ivp` in general.

Reply:
Thanks for clarifying the convention on the usage of backticks!

Your point about moving this documentation made me realize that I am probably using `options` wrong! As I am using `catch_exceptions` now, it changes the behavior of `solve_ivp`. It really should not be part of the `options` that get passed to the solver! So `catch_exceptions` should appear explicitly as in the signature of `solve_ivp`.

I will modify the code to address this.

Reply:
Ok... the most recent commit should fix this.